docs/aws/README.md

```diff
@@ -28,7 +28,7 @@ This command will create IAM resources used by Karpenter. We recommend using [Cl
 ```bash
 aws cloudformation deploy \
   --stack-name Karpenter-${CLUSTER_NAME} \
-  --template-file ./docs/aws/karpenter.cloudformation.yaml \
+  --template-file $(TEMPOUT=$(mktemp) | curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/v0.2.4/docs/aws/karpenter.cloudformation.yaml > $TEMPOUT | echo $TEMPOUT)  \
```

Comment:
I could be doing something wrong but I can't get this to worry (sorry if this is a red herring). I'm wondering if this might work? 

```bash
curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/v0.2.4/docs/aws/karpenter.cloudformation.yaml \
  | aws cloudformation deploy \
    --stack-name Karpenter-${CLUSTER_NAME} \
    --template-file /dev/stdin  \
   # the rest is the same...
```

Reply:
This didn't work for me @JacobGabrielson.

The following command works for me. IIUC pipes use different subshells, so the TEMPOUT variable will be saved in a different subprocess if you pipe it. You have to define it before.
```
TEMPOUT=$(mktemp)
curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/v0.2.4/docs/aws/karpenter.cloudformation.yaml > $TEMPOUT \
| aws cloudformation deploy \
  --stack-name Karpenter-${CLUSTER_NAME} \
  --template-file ${TEMPOUT}\
  --capabilities CAPABILITY_NAMED_IAM \
  --parameter-overrides ClusterName=${CLUSTER_NAME} OpenIDConnectIdentityProvider=$(aws eks describe-cluster --name ${CLUSTER_NAME} | jq -r ".cluster.identity.oidc.issuer" | cut -c9-)
```